index.html

```diff
@@ -0,0 +1,289 @@
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link text-light" href="#">News</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link text-light" href="#">projects</a>
+                </li>
+                <hr class="w-25" />
+                <li class="nav-item">
+                  <a href="./index.html" class="nav-link orangeColor"
+                    >Sign in</a
+                  >
+                </li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </nav>
+    </header>
+
+    <!-- hero section -->
+    <section class="hero">
+      <div class="container-fluid width8 hero-cont">
+        <p class="orangeColor hero1p">"IN PERSON. BY PHONE. ONLINE"</p>
+        <h1 class="orangeColor">
+          THE SMARTER WAY <br />TO FILE TAX <br />RETURN
+        </h1>
+        <p class="darkColor bg-light width5 border border-5 p-3">
+          Our certification programs give you the opportunity to gain the
+          highest-quality education in international tax and equip you with the
+          knowledge and skills you need to fulfill your role with confidence.
+        </p>
+        <h2 class="darkColor">2023.02.13(MON) ~ 24(FRIDAY)</h2>
+        <p class="darkColor fs-5">
+          @ City Tax Accountants Hall, Beirut, Lebanon
+        </p>
+      </div>
+    </section>
+
+    <!-- Tools & Applications section -->
+    <section class="toolsapp darkBgd text-light text-center">
+      <div class="container-fluid width8 py-5">
+        <h2>Tools & Applications</h2>
+        <div class="linesmall orangeColor orangeBgd mx-auto"></div>
+        <div class="row py-5">
+          <div class="col-md">
+            <div class="card bg-light bg-opacity-10">
+              <div class="card-body align-items-center">
+                <img src="./assets/images/tools/tools1.png" alt="tools 1" />
+                <h5 class="card-title orangeColor">Online Account</h5>
+                <p class="card-text">
+                  Access your individual account information including balance,
+                  payments, tax records and more.
+                </p>
+              </div>
+            </div>
+          </div>
+          <div class="col-md">
+            <div class="card bg-light bg-opacity-10">
+              <div class="card-body align-items-center">
+                <img src="./assets/images/tools/tools2.png" alt="tools 2" />
+                <h5 class="card-title orangeColor">Tax Record</h5>
+                <p class="card-text">
+                  Tax returns, tax account information, wage, and income
+                  statements, and verification of non-filing letters.
+                </p>
+              </div>
+            </div>
+          </div>
+          <div class="col-md">
+            <div class="card bg-light bg-opacity-10">
+              <div class="card-body align-items-center">
```

Comment:
- [ ] For this section kindly:
-  make sure your cards are square and have the same size :blue_square: , 
-  remember to add a hover effect that shows a white border for each of them. :recycle: 
- reduce the distance between them.

Your design:
![Yprogram](https://user-images.githubusercontent.com/97834160/210765004-dc9c4ecf-a4ee-47de-8dd7-bdb415f0a791.png)

 Behance design:
![Bprogram](https://user-images.githubusercontent.com/97834160/210765213-b961ae6b-d464-4b8f-951a-38750a8533b6.png)